packages/expo-image-picker/android/src/main/AndroidManifest.xml

```diff
@@ -20,4 +20,14 @@
                 android:resource="@xml/image_picker_provider_paths"/>
         </provider>
     </application>
+
+    <queries>
+      <intent>
+        <action android:name="android.media.action.IMAGE_CAPTURE" />
```

Comment:
I wonder if this would be beneficial to have some comment why it's here 🤔 What do you think? 🤔